PopApp/tsconfig.json

```diff
@@ -0,0 +1,76 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
+    // "incremental": true,                   /* Enable incremental compilation */
+    "target": "esnext",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
+    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
+    "lib": ["es6"],                           /* Specify library files to be included in the compilation. */
```

Comment:
Using `btoa` and `atob` result in a compilation error. Adding the DOM library to the tsconfig file like in [this post](https://stackoverflow.com/questions/42899209/ts2304cannot-find-name-btoa) solves the issue.

(I cannot "propose changes" directly in the PR. Could it be a permissions issue?)

Reply:
Done. 
I think the proposed changes being blocked is only due to the fact that the commit is "outdated", it's not a permission issue.